kivymd/uix/textfield.py

```diff
@@ -1036,7 +1036,7 @@ def on_line_color_focus(self, *args):
         if self.color_mode == "custom":
             self._update_colors(self.line_color_focus)
 
-    def on__hint_text(self, instance, value):
+    def on_hint_text(self, instance, value):
```

Comment:
This is not a typo - https://github.com/kivy/kivy/blob/master/kivy/uix/textinput.py#L2633

Reply:
Does it a mistake in typing or it's already correct?

Reply:
This is an overridden built-in method of the Kivy library textinput widget.

Reply:
:+1: